translations/round_structure.tex/he.po

```diff
@@ -85,6 +106,8 @@ msgid ""
 "\\hspace{1em}\\includegraphics[width=1.6\\linewidth]{\\art/minotaur.png}\n"
 "\n"
 msgstr ""
+"\\hspace{1em}\\reflectbox{\\includegraphics[width=1.6\\linewidth]{\\art/minotaur.png}}\n"
```

Comment:
```suggestion
"\\hspace{-1em}\\reflectbox{\\includegraphics[width=1.6\\linewidth]{\\art/minotaur.png}}\n"
```
Try something like this, IMO it's a little too far away from the edge?

Reply:
IDK, I don't see the difference 
original:
<img width="602" alt="image" src="https://github.com/Heegu-sama/Homm3BG/assets/39484961/600421fd-c42e-4037-8210-6aab913c1a53">

after your change:
<img width="606" alt="image" src="https://github.com/Heegu-sama/Homm3BG/assets/39484961/8b8949eb-b35b-4c68-9bbf-66865a8e3acc">

Reply:
OK, it looks different when you post the whole page. I really recommend the script for this. Is it really the case that `hspace` doesn't work? Can you try `\hspace{10em}`?

Reply:
Or maybe with Hebrew it needs to be after the Minotaur, not before.

Reply:
Argh, to the right of the Minotaur in LaTeX.

Reply:
Im still not sure if we want the Minotaur to be closer to the edge or closer to the center of the page